### PR TITLE
Feature/linkta 216 set up landing page

### PR DESCRIPTION
**Pull request by kinskiwu**
## Description
- created new folders and files according to new design
- populated new files with placeholder code

Fixes #Linkta-216 (issue)

## Type of change

- [X] ✨  New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- [X] Manual testing by running npm run dev to ensure components render and route setup working

## Checklist:

- [X] I have performed a self-review of my own code.
- [X] My changes generate no new warnings.
- [X] I have reviewed the [**Security Best Practices Document**](SECURITY_BEST_PRACTICES.md).
- [X] I have followed the [**Naming Conventions Guide**](NAMING_CONVENTIONS_GUIDE.md) for my changes.
- [X] I have ensured that my pull request title is descriptive.

## Additional Information:
Updated some component names according to the [design doc](https://docs.google.com/document/d/1q8gHnFBQaTkFE2aRuZeUbiaIg2or5xTttNLdcUt5C8w/edit) under 'Landing Page Repo'. Let me know if they are good to go @ChTiSh

### CODE REVIEW

File: pages/api/waiting-list/index.ts
Comment (ChTiSh) [inline pages/api/waiting-list/index.ts:1]: I think we have to name the endpoint either page.js or route.js:
https://nextjs.org/docs/app/building-your-application/routing
Also, .js, .jsx, or .tsx file extensions can be used for special files.

Comment (ChTiSh) [reply pages/api/waiting-list/index.ts:1]: assuming we are using app router

Comment (ChTiSh) [reply pages/api/waiting-list/index.ts:1]: I just realized it's using page router

File: app/layout.tsx
Comment (ChTiSh) [inline app/layout.tsx:1]: we might need to separate the root level layout and component level layout, as some of our component will not need the header and footer

**Review by LucieSeidler — APPROVED** [review]

**Review by ChTiSh — APPROVED** [review]

**Pull request merged**